visualization/CMakeLists.txt

```diff
@@ -212,26 +212,6 @@ if(HAVE_QVTK)
   endif()
 endif()
 
-set(EXT_DEPS "")
-if(WITH_OPENNI)
-  list(APPEND EXT_DEPS libopenni)
-endif()
-if(WITH_OPENNI2)
-  list(APPEND EXT_DEPS libopenni2)
-endif()
-if(WITH_ENSENSO)
-  list(APPEND EXT_DEPS ensenso)
-endif()
-if(WITH_DAVIDSDK)
-  list(APPEND EXT_DEPS davidSDK)
-endif()
-if(WITH_DSSDK)
-  list(APPEND EXT_DEPS dssdk)
-endif()
-if(WITH_RSSDK)
-  list(APPEND EXT_DEPS rssdk)
-endif()
-
 PCL_MAKE_PKGCONFIG(${LIB_NAME} COMPONENT ${SUBSYS_NAME} DESC ${SUBSYS_DESC} PCL_DEPS ${SUBSYS_DEPS} EXT_DEPS ${EXT_DEPS})
```

Comment:
The EXT_DEPS variable is now empty/undefined here. Is there a reason why vtk is not listed in there?